### PR TITLE
[Merged by Bors] - refactor(MultilinearMap): rename fields

### PR DESCRIPTION
**Pull request by urkud**
Use `map_update_*` instead of `map_*`, so that we can move these lemmas to the root namespace
once we introduce `MultilinearMapClass`.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by j-loreaux — COMMENTED** [review]
Aside from one name issue, this LGTM. After you address that I can send it to the maintainer queue.

File: Mathlib/LinearAlgebra/Multilinear/Basic.lean [hunk 1]
```diff
@@ -1100,7 +1112,7 @@ theorem map_smul_univ [Fintype ι] (c : ι → R) (m : ∀ i, M₁ i) :
   classical simpa using map_piecewise_smul f c m Finset.univ
 
 @[simp]
-theorem map_update_smul [DecidableEq ι] [Fintype ι] (m : ∀ i, M₁ i) (i : ι) (c : R) (x : M₁ i) :
+theorem map_smul_update [DecidableEq ι] [Fintype ι] (m : ∀ i, M₁ i) (i : ι) (c : R) (x : M₁ i) :
```
[the remark this answers is not in the document]
Comment (urkud) [reply Mathlib/LinearAlgebra/Multilinear/Basic.lean:1115]: WDYT about `map_update_smul_left`?

[the remark this answers is not in the document]
Comment (j-loreaux) [reply Mathlib/LinearAlgebra/Multilinear/Basic.lean:1115]: Okay

**Comment by j-loreaux** [comment]
maintainer merge

**Pull request closed without merging**